rdm/init_files/documents/software_plan.md

```diff
@@ -93,16 +94,22 @@ To the extent possible, checking against these standards should be performed in
 ## Testing Plan
 
 [[62304:5.5.2]]
+All final tests must include the git hash or other objective reference that can be used to identify the exact software tested [[62304:5.1.11]].
```

Comment:
Probably should add a blank line between this and the auditor note

Reply:
Fixed.